knowledge/laws/traffic_laws_us/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: traffic_laws
+seed_examples:
+- answer: | The maximum speed limit on rural two-lane roads ranges from 50 mph (80 km/h) in parts of the northeast to 75 mph (120 km/h) in parts of Texas.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!